src/julia-parser.scm

```diff
@@ -541,11 +547,14 @@
                    t)))))
 
 (define (take-token s)
-  (or
-   (begin0 (ts:pbtok s)
-           (aset! s 3 #f))
-   (begin0 (ts:last-tok s)
-           (ts:set-tok! s #f))))
+  (let ((t (or
+             (begin0 (ts:pbtok s)
+                     (aset! s 3 #f))
+             (begin0 (ts:last-tok s)
+                     (ts:set-tok! s #f)))))
+   (if (and (not allow-question-mark) (eq? t '?))
+       (syntax-deprecation s "`?` used as an identifier" ""))
```

Comment:
Ah thanks, I don't know how I missed that. It seems to work fine, I have updated the PR.